packages/plugin-core/src/test/suite-integ/CapitalizeSelectionCommand.test.ts

```diff
@@ -0,0 +1,49 @@
+import { NoteTestUtilsV4 } from "@dendronhq/common-test-utils";
+import path from "path";
+import * as vscode from "vscode";
+import { CapitalizeSelectionCommand } from "../../commands/CapitalizeSelectionCommand";
+import { VSCodeUtils } from "../../utils";
+import { expect, runMultiVaultTest } from "../testUtilsv2";
+import { setupBeforeAfter } from "../testUtilsV3";
+
+suite("CapitalizeSelectionCommand", function () {
+  let ctx: vscode.ExtensionContext;
+
+  ctx = setupBeforeAfter(this, {
+    beforeHook: () => {},
+  });
+  test("basic", (done) => {
+    runMultiVaultTest({
+      ctx,
+      preSetupHook: async ({ vaults, wsRoot }) => {
+        await NoteTestUtilsV4.createNote({
+          vault: vaults[0],
+          wsRoot,
+          fname: "gamma",
+          body: "hello world",
+        });
+      },
+      onInit: async ({ vaults, wsRoot }) => {
+        const notePath = path.join(wsRoot, vaults[0].fsPath, "gamma.md");
+        const editor = await VSCodeUtils.openFileInEditor(
+          vscode.Uri.file(notePath)
+        );
+
+        if (!editor) {
```

Comment:
since this is a test, we can just assert editor using an `as` operator

Reply:
1949b152